packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -0,0 +1,67 @@
+import { attr, observable } from '@microsoft/fast-element';
+import { DesignSystem, FoundationElement } from '@microsoft/fast-foundation';
+import { template } from '../theme-provider/template';
+import { styles } from './styles';
+import {
+    WaferMapColorsScale,
+    WaferMapDie,
+    WaferMapOrientation,
+    WaferMapQuadrant
+} from './types';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-wafer-map': WaferMap;
+    }
+}
+
+/**
+ * A nimble-styled WaferMap
+ */
+export class WaferMap extends FoundationElement {
+    @attr({
+        attribute: 'quadrant'
+    })
+    public quadrant: WaferMapQuadrant = WaferMapQuadrant.topLeft;
+
+    @attr({
+        attribute: 'orientation'
+    })
+    public orientation: WaferMapOrientation = WaferMapOrientation.top;
+
+    @attr({
+        attribute: 'max-characters'
+    })
```

Comment:
Since this attribute is a numeric value, we need to use a converter so that the string HTML attribute is converted to a numeric property

```suggestion
    @attr({
        attribute: 'max-characters',
        converter: nullableNumberConverter
    })
```

And that converter is available from fast element, ie:

```ts
import { nullableNumberConverter } from "@microsoft/fast-element";
```

Reply:
Added, thank you! :)